doc/Enable-GEOS-and-Proj4-on-Heroku.md

```diff
@@ -79,6 +83,8 @@ heroku plugins:install heroku-repo
 heroku repo:purge_cache
 ```
 
+**Note** In order for this to work [rgeo-proj4] you will need to use ENV vars to update to PROJ 6.2+.
```

Comment:
@earnold thanks for the clarification. It looks like from the buildpack code (README is a bit unclear on that point) that the way to make it work is:

```bash
echo 6.2 > PROJ_VERSION
git add PROJ_VERSION
git commit -m "Set proj version for geo-buildpack"
```

Would you have an ENV on which you could test that this works ? We'll avoid someone opening another issue on that exact topic!

And if it doesn't work, one will have to post an issue to the buildpack repo (I may do it).

Reply:
Happy to test - but might not get to it later this week / early next week.

Reply:
@earnold, that is fine for me. Let's say we merge asis in one month if it is stale. And someone will reopen an issue when they encounter the problem.

Reply:
@BuonOmo - I gave the buildpack a brief test and I couldn't get it working. 

One nitpick: to set the heroku env var, the command to use is `heroku config:set PROJ_VERSION=6.2`

Even with that, however, I get an error: 
```
remote: -----> Building on the Heroku-20 stack
remote: -----> Using buildpacks:
remote:        1. https://github.com/heroku/heroku-geo-buildpack.git
remote:        2. heroku/ruby
remote: -----> Geo Packages (GDAL/GEOS/PROJ) app detected
remote: -----> Installing GDAL-2.4.0
remote: -----> Installing GEOS-3.7.2
remote: -----> Installing PROJ-6.2
remote:  !     Requested PROJ Version (6.2) is not available for this stack (heroku-20).
remote:  !     Aborting.
```

I also tried with PROJ_VERSION set to 6.3.1 and 7.2.1, and those failed as well. 

I can't rule out that it would be possible to get it working, but it's definitely not working for me.

Reply:
@earnold thank you for your time! I think we have a great reference now on why this is not working with the geo buildpack, and if one where to ask we could point toward that PR and tell the to open an issue or fix on the buildpack.